src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  width: $card-size;
+  background: $color-main;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    @extend %font-default;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $card-h-color;
+  }
+
+  &__code {
+    @extend %font-default;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $card-p-color;
+
+    margin: 0;
+    margin-bottom: 16px;
```

Comment:
You can use shorthand properties instead
https://developer.mozilla.org/en-US/docs/Web/CSS/margin#syntax